src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/IssueCreateDialog.kt

```diff
@@ -70,10 +70,12 @@ class IssueCreateDialog(
      * Click 'Configure Fields' to display 'All' to ensure
      * all mandatory fields are displayed in creation dialog.
      *
+     * The view has changed since Jira 8.19 so differentiator is needed
```

Comment:
Would the new [semantic DOM attributes from 8.14](https://bulldog.internal.atlassian.com/browse/MNSTR-4635) cover this?

Reply:
It should since currently we are testing up to 8.13 (based on the ci.yml)

Reply:
The attributes linked above are a response to https://github.com/atlassian/jira-actions/pull/77
The "then new" GUI was detected via a very complicated and very unstable selector in `fun getVisibleAllFieldsLinkLocator`. Now we're adding "new new GUI". Just like  `New Folder (6)` on desktop or `essay-final3-LAST (2).doc` :wink:  There's always gonna be something newer, unless we future-proof it.

The future-proofing idea is: get a stable (covered by HTML tests :heavy_check_mark:) semantic (user terms, not tech :heavy_check_mark:) DOM and rely on that for the long term. These were shipped in 8.14. Semantic attributes should be detected first, then fall back to legacy non-semantic code (like `dl:nth-of-type(1)` and `/div[@class='$dialogClass']//dl[$sectionIndex]//dd[1]//a`).

The `jira-actions` didn't consume the attributes yet, so we're not sure if they work at all and if they'll work with 8.19.
Unfortunately, it's hard to test unreleased Jira versions in this module here, because these tests don't reuse the  `infrastructure` API, which would allow that. We'll get that fixed after I ship [JPERF-273](https://ecosystem.atlassian.net/browse/JPERF-273) in innovation time.

Reply:
> it's hard to test unreleased Jira versions in this module here

First EAP of 8.19 should be available soon (see Atlassian internal Slack on #jira-server-release).

Reply:
Thanks for clarifying, I definitely agree.

Reply:
So 8.14 future proofing isn't guaranteed to work straight away, so we can do that in a separate PR. This PR adds value of 8.19+ support and doesn't incur much tech debt. Imma approving :smiley: